lib/hq/graphql/association_loader.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
```

Comment:
This file is just copied from lib/hq/graphql/loaders/association.rb.

Not sure why the dif isnt showing that